native/package.json

```diff
@@ -7,13 +7,16 @@
     "start:integreat-test-cms": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' react-native start --reset-cache",
     "start:integreat": "cross-env BUILD_CONFIG_NAME='integreat' react-native start --reset-cache",
     "start:malte": "cross-env BUILD_CONFIG_NAME='malte' react-native start --reset-cache",
+    "start:abapp": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native start --reset-cache",
```

Comment:
I think we should use start:aschaffenburg here. At least I dont have a problem with the length.

Reply:
Also abapp is short for aschaffenburg app not aschaffenburg :sweat_smile: